### PR TITLE
Add Wix

### PR DESCRIPTION
**Pull request by barakplasma**
Wix.com is a website hosting platform, currently without 2FA, which helps users to collect payments from their customers.

### CODE REVIEW

File: _data/hosting.yml [hunk 1]
```diff
@@ -504,6 +504,14 @@ websites:
       tfa: No
       twitter: weebly
       facebook: weebly
+      
+    - name: Wix
+      url: https://www.wix.com/
+      twitter: wixhelp
```
Comment (Carlgo11) [inline _data/hosting.yml]:
```suggestion
      twitter: WixHelp
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I suggest you use this logo instead:
Wix

**Comment by barakplasma** [comment]
@Carlgo11 I made the changes your requested

Wix has a two factor authentication feature request webpage ( https://support.wix.com/en/article/request-two-factor-authentication-for-wix-accounts ) where user's can vote for 2FA, but I didn't see a place to put this into yaml file.

**Review by Carlgo11 — APPROVED** [review]

File: _data/hosting.yml [hunk 2]
```diff
@@ -504,6 +504,14 @@ websites:
...
       twitter: weebly
       facebook: weebly
+      
+    - name: Wix
+      url: https://www.wix.com/
+      twitter: WixHelp
+      facebook: wix
+      img: wix.png
+      tfa: No
+
```
Comment (Carlgo11) [inline _data/hosting.yml:514]: Looks like there's a new line at the bottom that shouldn't be there.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**